eZ/Bundle/EzPublishCoreBundle/Resources/views/fielddefinition_settings.html.twig

```diff
@@ -27,6 +27,13 @@
 </ul>
 {% endblock %}
 
+{% block ezrichtext_settings %}
+    <ul class="fielddef-settings {{ fielddefinition.fieldTypeIdentifier }}-settings">
+        {% set rows = settings.numRows %}
```

Comment:
actually we don't take into account this field settings and we also have an improvement about removing it in JIRA (https://jira.ez.no/browse/EZP-25007) so I guess we should "hide" this option there (until it's really removed)